Source/WebCore/rendering/AttachmentLayout.mm

```diff
@@ -199,6 +199,7 @@ static Color titleTextColorForAttachment(const RenderAttachment& attachment, Att
 #if PLATFORM(IOS_FAMILY)
 
 constexpr CGSize attachmentSize = { 160, 119 };
+constexpr CGSize attachmentIconOnlySize = { 56, 72 };
```

Comment:
Should probably leave a comment here and the in CSS file if the values are co-dependent.